website/lib/analytics/index.js

```diff
@@ -5,7 +5,9 @@ module.exports = {
 
   contentFor(type) {
     if (type === 'head-footer' && process.env.VERCEL_ENV === 'production') {
-      return `<script src="https://cdn.usefathom.com/script.js" data-spa="auto" data-site="PPQLJJKK" data-excluded-domains="localhost" defer></script>`;
+      return `<script src="https://cdn.usefathom.com/script.js" data-spa="auto" data-site="PPQLJJKK" data-excluded-domains="localhost" defer></script>
```

Comment:
Do we also want to exclude the vercel preview deploys?
```suggestion
      return `<script src="https://cdn.usefathom.com/script.js" data-spa="auto" data-site="PPQLJJKK" data-excluded-domains="vercel.app,localhost" defer></script>
```

Reply:
`process.env.VERCEL_ENV === 'production'` will exclude those and https://hds-website.vercel.app/ is now redirected to our main domain.

Reply:
ah, nice!

Reply:
can't hurt though 😄